rust/extractor/src/translate.rs

```diff
@@ -80,6 +156,671 @@ impl CrateTranslator<'_> {
             })
     }
 
+    fn emit_label(
```

Comment:
uh, I can see we could get in nomenclature confusion between trap labels and rust labels... maybe we should `s/label/trap_label` where it applies to avoid that?

Reply:
I'm not sure I understand . This function does not emit a trap label .

Reply:
Ah, sorry, my bad, I got confused: we actually don't have any function here named with `label` in it and referring to trap labels. All is good :slightly_smiling_face: